### ISSUE TITLE
Contact respective teams regarding icu build

### ISSUE THREAD

**Issue by ctrlaltf24** [FaithLife-Community/LogosLinuxInstaller#216]
Some teams/projects around the internet that may be interested in a clean/working icu build now that windows ships it
- [ ] [Bottles](https://docs.usebottles.com/contribute/missing-dependencies)
- [ ] [WineTricks](https://github.com/Winetricks/winetricks?tab=readme-ov-file#custom-verb-files)

**Comment by thw26** [comment]
Yes! Our plan is to supply upstream code to WineTricks for ICU, as that should make it easier for us, too.

**Comment by ctrlaltf24** [comment]
Mentioned current work in wine bug in case someone else on the internet needed this for another reason. Still would be nice to get to eventually https://bugs.winehq.org/show_bug.cgi?id=53354#c5

**Comment by thw26** [comment]
I believe this has been resolved upstream in Wine 11.x and we can close this issue as being resolved eventually?
